packages/runtime/runtime-utils/src/summarizerNode/summarizerNodeUtils.ts

```diff
@@ -69,10 +69,20 @@ export class EscapedPath {
     }
 }
 
+export interface ISummaryNode {
+    readonly referenceSequenceNumber: number;
```

Comment:
can you please add comments to all the fields, and some description on how they differ? (i.e. fullPath vs. fullPathForChildren)

Reply:
Removed this interface as its not needed